### PR TITLE
Pin openssl to 1.1.1d. Bump version to 0.2.1

### PR DESCRIPTION
**Pull request by fchorney**
Relevant openssl issues:
https://github.com/openssl/openssl/issues/11381
https://github.com/openssl/openssl/issues/11378

Pin openssl to 1.1.1d so we don't hit the openssl issues.

### CODE REVIEW

File: src/FTPServer.jl [hunk 1]
```diff
@@ -29,7 +29,7 @@ const PYTHON_CMD = joinpath(
 
 function __init__()
     Memento.register(LOGGER)
-
+    Conda.add("openssl==1.1.1d")
```
Comment (nicoleepp) [inline src/FTPServer.jl]: Should have a comment explaining why

Comment (nicoleepp) [reply src/FTPServer.jl]: Also maybe a link to an issue

Comment (omus) [reply src/FTPServer.jl]: This should be added into `deps/build.jl` so it only happens when the package is installed.

File: .travis.yml [hunk 2]
```diff
@@ -16,13 +16,12 @@ env:
...
   matrix:
     - CONDA_JL_VERSION="3"  # Documentation stage will use the first variable set
-    - CONDA_JL_VERSION="2"
 jobs:
   allow_failures:
     - julia: nightly
 
     # Missing packages on Windows
-    - env: CONDA_JL_VERSION="2"
+    - env: CONDA_JL_VERSION="3"
```
Comment (omus) [inline .travis.yml]: I think this allow failure was specific to Conda 2 on Windows so this can be removed.

File: .travis.yml [hunk 3]
```diff
@@ -16,13 +16,12 @@ env:
     - PYTHON=""
   matrix:
     - CONDA_JL_VERSION="3"  # Documentation stage will use the first variable set
-    - CONDA_JL_VERSION="2"
```
Comment (omus) [inline .travis.yml]: Sorry to be a pain but I think it's worth removing Conda 2 support as a separate PR

Comment (fchorney) [reply .travis.yml]: ok I've reverted the change

**Comment by omus** [comment]
CI failure is due `CONDA_JL_VERSION="2"` running on platforms which are not 32-bit.

**Pull request merged**